onnx/onnxifi_dummy.c

```diff
@@ -160,7 +160,8 @@ onnxGetExtensionFunctionAddress(
   }
   int count = 0;
   *function = NULL;
-  for (int i = 0; i < extension_function_number; i++) {
+  int i;
```

Comment:
what's this :P

Reply:
It is because I just realize C lang does not accept "for (int i = 0; ...)".... T^T

Reply:
Hmm, what's wrong with that?